config/config.go

```diff
@@ -111,6 +117,140 @@ func GetNodeConfig(nodeConf *NodeConfig, cmConf *cmcfg.Config) {
 	}
 }
 
+// UpdateNodeConfigWithFlags updates NodeConfig with values from command line flags.
+func UpdateNodeConfigWithFlags(nodeConf *NodeConfig, flags *pflag.FlagSet) (err error) {
+	updateBoolFlag := func(flagName string, updateFunc func(bool)) {
+		if flag := flags.Lookup(flagName); flag.Changed {
+			updateFunc(flag.Value.String() == "true")
```

Comment:
I think it would be good touse [`strconv.ParseBool`](https://pkg.go.dev/strconv#ParseBool) for better UX.